vtm-playground/src/org/oscim/test/MarkerLayerLabelsTest.java

```diff
@@ -180,7 +180,7 @@ private MarkerSymbol createAdvancedSymbol(MarkerItem item, Bitmap poiBitmap) {
         // titleCanvas.fillRectangle() does not support transparency
         titleCanvas.drawCircle(0, 0, xSize * 2, fillPainter);
 
-        titleCanvas.drawText(item.title, margin, titleHeight - margin, textPainter);
+        titleCanvas.drawText(item.title, margin, titleHeight - margin - textPainter.getFontDescent(), textPainter);
```

Comment:
> This fixes the fact that font decends are cut away.

Do you have a text example that we could test the cut?

Reply:
For example the "Brandenburger Tor" seems rendered at center without cuts.

Reply:
Now that is strange. Any text with a lowercase G gets the descent cut away. 

I played around a bit with the code, but it should not have changed the original parts that would mess the descend.

My current testbed uses for the creation of the symbol: 

```
    /**
     * Creates a transparent symbol with text and description.
     *
     * @param item      -> the MarkerItem to process, containing title and description
     *                  if description starts with a '#' the first line of the description is drawn.
     * @param poiBitmap -> poi bitmap for the center
     * @return MarkerSymbol with title, description and symbol
     */
    private MarkerSymbol createAdvancedSymbol(MarkerItem item, Bitmap poiBitmap) {
        final Paint textPainter = CanvasAdapter.newPaint();
        textPainter.setStyle(Paint.Style.FILL);
        textPainter.setColor(FG_COLOR);
        int bitmapHeight = poiBitmap.getHeight();
        textPainter.setTextSize(bitmapHeight);
        textPainter.setTypeface(Paint.FontFamily.MONOSPACE, Paint.FontStyle.NORMAL);

        final Paint haloTextPainter = CanvasAdapter.newPaint();
        haloTextPainter.setStyle(Paint.Style.FILL);
        haloTextPainter.setColor(Color.WHITE);
        haloTextPainter.setTextSize(bitmapHeight);
        haloTextPainter.setTypeface(Paint.FontFamily.MONOSPACE, Paint.FontStyle.BOLD);

        int margin = 3;
        int dist2symbol = (int) Math.round(bitmapHeight * 1.5);

        int titleWidth = ((int) haloTextPainter.getTextWidth(item.title) + 2 * margin);
        int titleHeight = (int) (haloTextPainter.getTextHeight(item.title) + textPainter.getFontDescent() + 2 * margin);

        int symbolWidth = poiBitmap.getWidth();

        int xSize = Math.max(titleWidth, symbolWidth);
        int ySize = titleHeight + symbolWidth + dist2symbol;

        // markerCanvas, the drawing area for all: title, description and symbol
        Bitmap markerBitmap = CanvasAdapter.newBitmap(xSize, ySize, 0);
        org.oscim.backend.canvas.Canvas markerCanvas = CanvasAdapter.newCanvas();
        markerCanvas.setBitmap(markerBitmap);

        // titleCanvas for the title text
        Bitmap titleBitmap = CanvasAdapter.newBitmap(titleWidth + margin, titleHeight + margin, 0);
        org.oscim.backend.canvas.Canvas titleCanvas = CanvasAdapter.newCanvas();
        titleCanvas.setBitmap(titleBitmap);

        titleCanvas.drawText(item.title, margin, titleHeight - margin - textPainter.getFontDescent(), haloTextPainter);
        titleCanvas.drawText(item.title, margin, titleHeight - margin - textPainter.getFontDescent(), textPainter);

        markerCanvas.drawBitmap(titleBitmap, xSize * 0.5f - (titleWidth * 0.5f), 0);
        markerCanvas.drawBitmap(poiBitmap, xSize * 0.5f - (symbolWidth * 0.5f), ySize * 0.5f - (symbolWidth * 0.5f));

        return (new MarkerSymbol(markerBitmap, MarkerSymbol.HotspotPlace.CENTER, true));
    }
```
When I didn't have the getFontDescent parts added, everything was cut. Can you try to enlarge the size in your example? Maybe it was that?

PS: ignore the double textPaint, it was a try to have a halo like feeling.

Reply:
With the PR and font size 30 there are cuts on top, e.g. "Potsdamer Platz":

![VTM](https://user-images.githubusercontent.com/3484020/55480630-69315780-5629-11e9-8c1e-8f8472792602.png)

Reply:
You are right, I had not pushed that part, sorry for that. It should now be in this PR. The title canvas needs to be enlarged accordingly.